script.js

```diff
@@ -1,66 +1,462 @@
+			emptyModalParent.addEventListener("click", close_modal);
+			return;
+		}
+
+		list.push({
+			id: nanoid(),
+			description: input.value.trim(),
+			taskCompleted: false,
+			date: date,
+			editMode: false
+		});
+		input.value = "";
+	}
+	else {
+		list.push({ ...listItem, editMode: false });
+	}
+	if (list.length === 1) {
+		let emptyImage = _("emptyImage");
+		if (emptyImage) {
+			emptyImage.remove();
+			_("list_items").classList.remove("addEmptyImage");
+			_("searchText").classList.remove("noshow");
+		}
+	}
+	let newItem = "<li id='item" + list[list.length - 1].id + "' ><div class='list_content'>" +
+		"<input " + (list[list.length - 1].taskCompleted ? "checked" : "") + " type='checkbox' class='checkbox' id='check-" + list[list.length - 1].id + "'>" +
+		"<span class='descriptionAndDate'><span id = 'desc-" + list[list.length - 1].id + "' class='description " + (list[list.length - 1].taskCompleted ? "checkboxChecked" : "") + "'>" + list[list.length - 1].description + "</span>" +
+		"<div class='descriptionDate'>" + list[list.length - 1].date + "</div></span>" +
+		"</div>" +
+		"<div class='dropdown'>" +
+		"<a id='menu-" + list[list.length - 1].id + "'>" + "<img class='dotmenu' id='dotmenu-" + list[list.length - 1].id + "' src='./images/dotmenu.svg'>" + "</a>" +
+		"</div>" +
+		"<div class='overlay' id='overlay'>" +
+		"<button id='alter-" + list[list.length - 1].id + "' class='alter_button'>" + "Edit" + "</button>" +
+		"<button id='delete-" + list[list.length - 1].id + "' class='delete_button'>" + "Delete" + "</button>" +
+		"<button id='cancel-" + list[list.length - 1].id + "' class='cancel_button'>" + "Cancel" + "</button>" +
+		"</div>" +
+		"<div class='buttons' id='buttons'>" +
+		"<button id='edit-" + list[list.length - 1].id + "' class='edit_button'><img class='editButtonImage' id='editimage" + list[list.length - 1].id + "' src='./images/edit.png'></button>" +
+		"<button id='remove-" + list[list.length - 1].id + "' class='remove_button'><img id='removeimage" + list[list.length - 1].id + "' src='./images/icons8-trash-can-50.png'></button>" +
+		"</div></li>";
+	_("printing_list").insertAdjacentHTML('afterbegin', newItem);
+
+	let menu = _("menu-" + list[list.length - 1].id);
+	let dotmenu = _("dotmenu-" + list[list.length - 1].id);
+	function displayButtons(e) {
+		e.stopPropagation();
+		_("overlay").classList.add("flexDisplay");
+		_("floatDiv").classList.add("floatDiv");
+		_("floatButton").classList.add("noshow");
+	}
+	menu.addEventListener("click", displayButtons);
+	dotmenu.addEventListener("click", displayButtons);
+
+	let checkBox = _("check-" + list[list.length - 1].id)
+	let description = _("desc-" + list[list.length - 1].id)
+
+	//cross-out on clicking description or checkbox
+	checkBox.addEventListener("change", (e) => {
+		let targetId = e.target.id.replace("check-", "")
+		if (list.find(element => {
+			return element.id.toString() === targetId
```

Comment:
No need to use the `return` statement if there's only a single line of code in arrow functions. Let's make this more concise and clean. 

```suggestion
		if (!list.find(element => element.id.toString() === targetId).editMode) {
```

Reply:
Update the same everywhere else in the code.